packages/build-lz-options/src/utilities/prompts.ts

```diff
@@ -93,6 +93,7 @@ const promptForNativeDropAddress: PromptObject<'nativeDropAddress'> = {
     name: 'nativeDropAddress',
     message: 'What native gas drop do you want to set?',
     initial: makeBytes32(),
+    validate: (str: string) => str.length <= 66, // 32 bytes is 64 nibbles + '0x'
```

Comment:
`makeBytes32` will throw if passed an invalid `bytes32` value - you can use that to validate this (by `try/catch`)

Reply:
good point, will do.